recipes/sentry-native/all/conanfile.py

```diff
@@ -87,6 +87,8 @@ def configure(self):
             del self.options.fPIC
 
     def requirements(self):
+        self.requires("openssl/1.1.1m")
+
```

Comment:
why?

Reply:
There is a conflict between cmake dependency and some other dependency.

Reply:
It should be solved with another solution (version conflict between requirements & build requirements is a bug of conan v1 with 1 profile), a CCI recipe should not lie on direct dependencies.

Reply:
What is the proper solution then? :)

Reply:
Bump openssl to 1.1.1m in offending recipe.

Reply:
it used to be in the line 104 to resolve qt dependencies conflicts. Why it is acceptable in that case?

Reply:
> it used to be in the line 104 to resolve qt dependencies conflicts. Why it is acceptable in that case?

it is not, it should be removed as well